app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/projectManagerBackendApi.ts

```diff
@@ -0,0 +1,153 @@
+/** @file Module containing the API client for the local backend API.
+ *
+ * Each exported function in the {@link ProjectManagerBackendAPI} in this module corresponds to an API endpoint. The
+ * functions are asynchronous and return a `Promise` that resolves to the response from the API. */
```

Comment:
I was chatting with Pawel here. Withotu this chat I was totally confused what is this file about and what is the difference between "Project Manager Backend" and "Project Manager". This should be named probably "localBackend", so it will nicely fit the "cloudBackend" name. 

Also, please double check if these docs are correct, as I think they refer improper files.